src/core/build_env.go

```diff
@@ -159,6 +159,7 @@ func TestEnvironment(state *BuildState, target *BuildTarget, testDir string) Bui
 		// We shouldn't really have specific things like this here, but it really is just easier to set it.
 		"GTEST_OUTPUT=xml:"+resultsFile,
 		"PEX_NOCACHE=true",
+		fmt.Sprintf("_TEST_RUN=%d", run),
```

Comment:
Could make this meaningless by hashing it and calling it something different.

Reply:
Good idea. Done.
Ideally I'd make it specific to the hash of the build action but I think this is sufficient.